Johnsona Procedure 2 PR - Replacing Wi-Fi Router

alyssajohnsonunofficial opened the pull request:
# Procedure 2 - Replacing a Wi-Fi Router

I began my second procedure, which is for medium-level users who want to replace their old Synology WRX560 Wi-Fi router with a new one. 


**Completed Work**
- Started on my second procedure: 'replacewifirouter' 
- Procedure's goal: To successfully uninstall an old Synology WRX560 Wi-Fi router and replace it with a new one. 
- I laid out the basic sections with headers, added some text and an image (I may look for a second one later), and made an empty numbered list for my steps to go into. 


# User Scenario - Experienced Installer, First-Time Replacer

**Scenario**: "Brittany is a wife and mother in her mid-30s. She wants to replace her old Synology WRX560 with a new one because the old one is beginning to slow down and can't keep up with her family's internet needs. She's experienced with the installation process, but not with the replacement process, and wants a helpful guide on how to switch out the old router for the new one as quickly and smoothly as possible."


## Notable Review Requests
- I tried to make the introduction more user-focused by using "your"; i.e. "your Wi-Fi router" instead of "a Wi-Fi router". Does this work better? If so, I'll change my first procedure to adopt this as well. 
- Because this procedure is for a slightly more advanced audience than the previous (with the intended user being someone who has installed a Synology router in the past), I didn't include a "terms to know" section or any subsections of the steps. Does this feel like a good match for the intended audience, or does it feel too difficult/complex? 
- Same questions of quality as before; is the intro too quick, how does the layout look? I'm still a little limited on photos but hoping to find more when I get more time to work on this.

Eden-Miller commented:
Hi Lauren! 

I know we've talked about this procedure a lot during our topic modeling work, but I'll put some of that information here for future use and reference. 

- Like we adopted for our topic model, I think it would be beneficial to break up this procedure into two separate procedures: "How to Request LinkedIn Admin Access" and "How to Assign LinkedIn Admin Statue."
- The procedure could benefit from having a clear "Introduction" and/or "Prerequisite" section to clearly display your staging moves. Like on the last procedure I reviewed, your beginning sentence works as a hybrid between an alerting step and a staging step. It may be better to have clear formatting to differentiate the two.  
- Some steps are a little wordy - try to clean them up. :)
- Your images are very efficient coaching moves! They're great. Labeling your figures could be helpful as well. 
- I don't see clear alerting moves in this procedure. It would helpful to highlight specific pain points so the audience can avoid them. I need to ask Dr. Lindgren about adding alerting moves within our task topics and what that would look like formatting wise. I'll add it to our project tracker. 

Great work, Lauren! I'm excited to continue to work with your procedures to complete our mini topic model.

Thank you!
Eden Miller

alyssajohnsonunofficial committed:
Created files

alyssajohnsonunofficial committed:
Create Procedure 2 + PR

dmbrasw2 commented on johnsona/PR-replace-wifirouter.md:
```diff
@@ -0,0 +1,22 @@
+
+
+## Notable Review Requests
+- I tried to make the introduction more user-focused by using "your"; i.e. "your Wi-Fi router" instead of "a Wi-Fi router". Does this work better? If so, I'll change my first procedure to adopt this as well. 
```
I think 2nd person language (you/your) is a good call for this

dmbrasw2 commented on johnsona/PR-replace-wifirouter.md:
```diff
@@ -0,0 +1,22 @@
+
+## Notable Review Requests
+- I tried to make the introduction more user-focused by using "your"; i.e. "your Wi-Fi router" instead of "a Wi-Fi router". Does this work better? If so, I'll change my first procedure to adopt this as well. 
+- Because this procedure is for a slightly more advanced audience than the previous (with the intended user being someone who has installed a Synology router in the past), I didn't include a "terms to know" section or any subsections of the steps. Does this feel like a good match for the intended audience, or does it feel too difficult/complex? 
```
I think it's a good match for a more advanced audience.

And theoretically these procedures would/could be included in the same documentation, so they could always refer to the previous 'terms to know' section if needed

gmsimon22 commented on johnsona/replacewifirouter.md:
```diff
@@ -0,0 +1,20 @@
+# How To Replace a Synology WRX560 Wi-Fi Router
+
+This procedure will guide you through the process of replacing your old Synology WRX560 Wi-Fi router with a new one. 
```
I think you can omit the term 'procedure' for this, since the audience sees it as a guide or manual. I like the more personable use of second person

WBlan97 replied:
I think you could include a few more staging moves for this one because the procedure is for a medium level user.

Instead of procedure I would say "set of instructions."

gmsimon22 commented on johnsona/replacewifirouter.md:
```diff
@@ -0,0 +1,20 @@
+- An ISP Modem (current or replacement)
+- An AC Power Adapter
+- An RJ-45 LAN Cable
+- A working outlet 
```
You might be able to remove these last two points, especially since they are an intermediate audience and they would hopefully know to use a working outlet

gmsimon22 commented on johnsona/replacewifirouter.md:
```diff
@@ -0,0 +1,20 @@
+
+![A Synology WRX560 Wi-Fi Router](./../johnsona/assets/images/Synology-WRX560-1-1536x864.jpg)
+
+## The Procedure
```
I liked how you sectioned out the installation procedure. I'm guessing that this is just a placeholder though for the subheadings.

gmsimon22 commented on johnsona/replacewifirouter.md:
```diff
@@ -0,0 +1,20 @@
+
+This procedure will guide you through the process of replacing your old Synology WRX560 Wi-Fi router with a new one. 
+
+## What You'll Need
```
I wouldn't mind adding a terms to know section in this procedure too, but maybe that's just because I would probably try to replace an existing machine without knowing anything about it. Not having this section seems appropriate though too for your intended audience

gmsimon22 reviewed (commented):
Nice start! I left my reviews.

WBlan97 commented:
I think this will end up being a very useful procedure!

dmbrasw2 reviewed (commented):
Looks great so far! Good pictures, and approachable writing

alyssajohnsonunofficial committed:
Plussed up the section names, image position and "What You'll Need" section

I moved my image to the top of the page, added a figure and caption, adjusted the "What You'll Need" section, and changed the name of my instructions

alyssajohnsonunofficial committed:
Quick change to the first entry in my "What You'll Need" section

I made a small change to the first entry in my list for the "What You'll Need" section for clarity

alyssajohnsonunofficial committed:
Revamped my intro

I changed my intro to be longer and more tailored to my audience

alyssajohnsonunofficial committed:
Added My Steps

I added the steps to my procedure, which include alerts

alyssajohnsonunofficial committed:
Adjusted "What You'll Need" section

alyssajohnsonunofficial committed:
Adjusted intro

alyssajohnsonunofficial committed:
Added a screenshot and copied and pasted its link into my screenshots folder

I added a screen capture that showed a close-up of the port of a router, and then I copied and pasted the code for it into my screenshots folder for my own reference

alyssajohnsonunofficial committed:
Added my user scenario

I made a copy of the user-scenario.md file and added my user scenario to it

alyssajohnsonunofficial committed:
Finished my "Wrap Up" section

I revamped and finished up my "Wrap Up" section at the end

alyssajohnsonunofficial committed:
Added a figure to my second image

I added a figure and caption to my screenshot

alyssajohnsonunofficial committed:
Adjusted intro one final time

alyssajohnsonunofficial committed:
Added an alert

I added an alert under my "What You'll Need" section

alyssajohnsonunofficial committed:
Finishing Touches

I fixed my link!

alyssajohnsonunofficial committed:
Adjusted wording in one of my alerts

alyssajohnsonunofficial committed:
Made My Reflection File, Added Headers and Images for My Reflection

I created a file for my 500 word reflection, added headers as per the instructions, and uploaded the screenshotted images I'm using to support some of my claims

alyssajohnsonunofficial committed:
Added my reflection

lingeringcode committed:
Feedback from Lindgren
